Source/Mosa.Utility.BootImage/VDI.cs

```diff
@@ -9,6 +9,8 @@ namespace Mosa.Utility.BootImage
 	/// </summary>
 	public static class VDI
 	{
+		// TODO: FIXME!!
```

Comment:
would be better to have `// FIXME: <comment>` to be more specific about what needs to be fixed